modules/lead_processor/src/restate/services/Lead/LeadVirtualObject.ts

```diff
@@ -124,7 +127,7 @@ export const LeadVirtualObject = restate.object({
 						);
 					}
 					const Lead = leadCreateRequest.data;
-					ShouldRunSync = Lead.SyncImmediately ?? true;
+					ShouldRunSync = leadType === LeadType.ACTON ? false : (Lead.SyncImmediately ?? true);
```

Comment:
Still allow the SyncImmediately parameter to run sync if the user passes it via API
```suggestion
					ShouldRunSync = Lead.SyncImmediately ?? leadType === "ACTON";
```